sim/dex.ts

```diff
@@ -1596,3 +1533,13 @@ dexes['base'] = new ModdedDex(undefined, true);
 dexes[BASE_MOD] = dexes['base'];
 
 export const Dex = dexes['base'];
+export namespace Dex {
+	export type Species = import('./dex-species').Species;
+	export type Item = import('./dex-items').Item;
+	export type Move = import('./dex-moves').Move;
+	export type Ability = import('./dex-abilities').Ability;
+
+	export type HitEffect = import('./dex-moves').HitEffect;
+	export type SecondaryEffect = import('./dex-moves').SecondaryEffect;
+	export type RuleTable = import('./dex-formats').RuleTable;
+}
```

Comment:
Am very annoyed I have to do this but I don't see any other way. TypeScript being TypeScript... :/